operations/helm/tests/test-enterprise-configmap-values-generated/mimir-distributed/charts/minio/templates/deployment.yaml

```diff
@@ -26,39 +28,48 @@ spec:
         app: minio
         release: test-enterprise-configmap-values
       annotations:
-        checksum/secrets: dc5f87186fbd122d4616e916d60a0c95a6939d0e3e70914edfc0362ee64a5c2c
-        checksum/config: a93cb8a58b08a5d5b8889b99ed92fa3ac7dde37d1ca5413212ba7fbd5ec02b13
+        checksum/secrets: 2cc5b473628374d82e197aa3a5c2ec68635bdb8f25b0b6895526ab1842502ff4
+        checksum/config: bbc2ca02c7c3016d6da9fbb104070191958d04d5c9c940162d59a9cb2c1aaf75
     spec:
-      serviceAccountName: "test-enterprise-configmap-values-minio"
       securityContext:
         runAsUser: 1000
         runAsGroup: 1000
         fsGroup: 1000
+        fsGroupChangePolicy: OnRootMismatch
+
+      serviceAccountName: minio-sa
       containers:
         - name: minio
-          image: "minio/minio:RELEASE.2021-02-14T04-01-33Z"
+          image: "quay.io/minio/minio:RELEASE.2022-05-08T23-50-31Z"
```

Comment:
We should mention this change in the changelog and have `minio.image.repository` in the values.yaml where it can be changed. The reason is that some enterprises don't allow using external repositories.